tests/main/snapshot-basic/task.yaml

```diff
@@ -29,9 +33,22 @@ execute: |
     # and is valid
     snap check-snapshot "$SET_ID"
 
+    # and is exportable
+    snap export-snapshot "$SET_ID" "${SET_ID}_export.snapshot"
+    test -e "${SET_ID}_export.snapshot"
+    test "$( tar -tvf "${SET_ID}_export.snapshot" | wc -l )" = "3"
+    # do it again and ensure it has the files have the same size
+    snap export-snapshot "$SET_ID" "${SET_ID}_export-2.snapshot"
+    s1="$(stat -c%s "${SET_ID}_export.snapshot" )"
+    s2="$(stat -c%s "${SET_ID}_export-2.snapshot" )"
+    test "$s1" = "$s2"
```

Comment:
How about just `diff "${SET_ID}_export.snapshot" "${SET_ID}_export-2.snapshot"` + a check that they have non-zero size?

OTOH it would be nice to check that the exported snapshot is valid (has canary files etc).

Reply:
Unfortunately the diff will not be the same because tar includes the creation timestamp. We could compare the file content though but the key part of this test bit is just that the size does not change on subsequent exports (which is a bit strange I admit :)

Reply:
Ah, indeed, I missed that, thanks for explanation!